<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelList</class>
 <widget class="QWidget" name="ModelList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>816</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>59999</width>
    <height>41500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(224, 252, 227);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <widget class="QPushButton" name="ApplyButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>修改</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="ModelList_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>切片厚度</string>
    </property>
    <property name="text">
     <string> Thickness(μm)：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="thicknesscombo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>160</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>130</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>切片中</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>184</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>     X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>184</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>     Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>184</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>     Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>202</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>尺寸</string>
    </property>
    <property name="text">
     <string>Size(mm):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelsizex">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>202</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelsizey">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>202</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelsizez">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>202</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>39</height>
     </size>
    </property>
    <property name="toolTip">
     <string>位置</string>
    </property>
    <property name="text">
     <string>Position(mm)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posx">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>228</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posy">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>228</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posz">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>228</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>258</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>选择</string>
    </property>
    <property name="text">
     <string>Rotation：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotx">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>254</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roty">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>254</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotz">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>254</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>286</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>缩放</string>
    </property>
    <property name="text">
     <string>Scaling：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scalex">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>280</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scaley">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scalez">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>280</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>83</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>删除</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/B9Edit/icons/delete2.png</normaloff>:/B9Edit/icons/delete2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>83</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>添加</string>
    </property>
    <property name="statusTip">
     <string>789</string>
    </property>
    <property name="whatsThis">
     <string>123</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="icon">
     <iconset resource="b9edit/b9edit.qrc">
      <normaloff>:/B9Edit/icons/new4.png</normaloff>
      <normalon>:/B9Edit/icons/delete3.jpg</normalon>
      <disabledoff>:/B9Edit/icons/delete2.png</disabledoff>:/B9Edit/icons/new4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateButton">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>复制</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/B9Edit/icons/new3.png</normaloff>:/B9Edit/icons/new3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="b9edit/b9edit.qrc"/>
 </resources>
 <connections/>
</ui>
